### DISCUSSION TITLE
Small, Large, and Dynamic viewport units

### DISCUSSION THREAD

**Discussion by MarcoMicale** [tailwindlabs/tailwindcss#10657] [Ideas]
Replace h-screen (height: 100vh;) with  h-screen (height: 100svh;)
Replace min-h-screen (min-height: 100vh;) with  min-h-screen (min-height: 100svh;)
Replace max-h-screen (max-height: 100vh;) with  max-h-screen (max-height: 100svh;)

**Comment by longmauriceproj** [comment]
or just add new classes to support dvh, lvh, and svh. I just wanted to bump this discussion, though I don't know how much visibility these get

**Comment by djfarly** [comment]
See: https://github.com/tailwindlabs/tailwindcss/pull/11317

As an alternative:

```js
const {
  height,
  minHeight,
  maxHeight
} = require('tailwindcss/defaultTheme');

/** @type {import('tailwindcss').Config} */
module.exports = {
  theme: {
    extend: {
      height: {
        ...height,
        screen: '100svh'
      },
      minHeight: {
        ...minHeight,
        screen: '100svh'
      },
      maxHeight: {
        ...maxHeight,
        screen: '100svh'
      },
    },
  },
};
```
